alert-database/src/main/java/com/synopsys/integration/alert/database/job/JobConfigurationModelFieldPopulationUtils.java

```diff
@@ -0,0 +1,201 @@
+                                                             .stream()
+                                                             .map(BlackDuckJobNotificationTypeEntity::getNotificationType)
+                                                             .collect(Collectors.toList());
+        blackDuckConfigurationModel.put(createConfigFieldModel("provider.distribution.notification.types", blackDuckJobNotificationTypeNames));
+
+        List<BlackDuckJobPolicyFilterEntity> blackDuckJobPolicyFilters = blackDuckJobDetails.getBlackDuckJobPolicyFilters();
+        if (null != blackDuckJobPolicyFilters && !blackDuckJobPolicyFilters.isEmpty()) {
+            List<String> blackDuckPolicyNames = blackDuckJobPolicyFilters
+                                                    .stream()
+                                                    .map(BlackDuckJobPolicyFilterEntity::getPolicyName)
+                                                    .collect(Collectors.toList());
+            blackDuckConfigurationModel.put(createConfigFieldModel("blackduck.policy.notification.filter", blackDuckPolicyNames));
+        }
+
+        List<BlackDuckJobVulnerabilitySeverityFilterEntity> blackDuckJobVulnerabilitySeverityFilters = blackDuckJobDetails.getBlackDuckJobVulnerabilitySeverityFilters();
+        if (null != blackDuckJobVulnerabilitySeverityFilters && !blackDuckJobVulnerabilitySeverityFilters.isEmpty()) {
+            List<String> blackDuckJobVulnerabilitySeverityNames = blackDuckJobVulnerabilitySeverityFilters
+                                                                      .stream()
+                                                                      .map(BlackDuckJobVulnerabilitySeverityFilterEntity::getSeverityName)
+                                                                      .collect(Collectors.toList());
+            blackDuckConfigurationModel.put(createConfigFieldModel("blackduck.vulnerability.notification.filter", blackDuckJobVulnerabilitySeverityNames));
+        }
+    }
+
+    public static void populateChannelConfigurationModelFields(DistributionJobEntity jobEntity, ConfigurationModelMutable channelConfigurationModel) {
+        String channelDescriptorName = jobEntity.getChannelDescriptorName();
+        channelConfigurationModel.put(createConfigFieldModel("channel.common.enabled", jobEntity.getEnabled().toString()));
+        channelConfigurationModel.put(createConfigFieldModel("channel.common.name", jobEntity.getName()));
+        channelConfigurationModel.put(createConfigFieldModel("channel.common.channel.name", channelDescriptorName));
+        channelConfigurationModel.put(createConfigFieldModel("channel.common.frequency", jobEntity.getDistributionFrequency()));
+        channelConfigurationModel.put(createConfigFieldModel("provider.distribution.processing.type", jobEntity.getProcessingType()));
+
+        if ("channel_azure_boards".equals(channelDescriptorName)) {
```

Comment:
These descriptor names should be coming from the channel keys or static strings in the channel keys.